crates/modelardb_compression_python/tests/__init__.py

```diff
@@ -0,0 +1,51 @@
+import unittest
```

Comment:
I could not find any argument against having tests in `__init__.py` so I think it is fine for now since we have so few tests written in Python. However, I agree that we should move the tests to separate files if we significantly expand the number of tests.